learning-bitmark/quick-start/working-with-bitmarks/README.md

```diff
@@ -1,11 +1,23 @@
-# Working with bitmarks
+# Working with Bitmarks
 
-## [Creating Bitmark Account](creating-bitmark-account.md#creating-bitmark-account)
+Interacting with Bitmark requires the creation of a Bitmark Account. Afterward, users can register Bitmark Certificates for their digital properties, which they may later transfer to other users or divide into Bitmark shares.
+
+Most work with the Bitmark Property System can be done via one of three methods: the [Bitmark App](https://a.bitmark.com/) (for iOS or Android), the [Bitmark SDK](https://github.com/bitmark-inc) (for a variety of programming languages), or the Bitmark CLI (available as part of the [Bitcoin node](https://github.com/bitmark-inc/docs/blob/master/learning-bitmark/quick-start/simple-solution-for-node-setup.md). 
+
```

Comment:
> or the Bitmark CLI (available as part of the [Bitcoin node]
Do you mean `Bitmark node`? And it misses a ')'

Reply:
Yep, thanks.